### DISCUSSION TITLE
sktime for time series LASSO regression

### DISCUSSION THREAD

**Discussion by LauraWeile** [alan-turing-institute/sktime#1317] [Q&A]
Hi! 
I tried reading through the documentation of sktime as I want to run a LASSO regression on time series data. I have several features in my data set including lagged observations of the variable that I want to forecast. 
I cannot find anything on LASSO regression in the documentation, but will sktime support the sklearn LASSO as a regressor? If not, do anyone have a suggestion to how I proceed from here? 
Thank you!

**Comment by fkiraly** [comment]
@LauraWeile, going currently through the questions and found this one unanswered.

What you want to do can be done using `sklearn`'s `Lasso` or `LassoCV`, together with the `sktime` reduction tools in `make_reduction`.

For details, look at the forecasting tutorial and change the estimator in the "reduction" example to one of the lassos.
